docs/testing-overview.md

```diff
@@ -16,6 +31,181 @@ Code style in JavaScript is enforced using [ESLint](http://eslint.org/). The abo
 
 To run unit tests only, without the linter, use `npm run test-unit` instead.
 
+### Folder structure
+
+Keep your tests in a `test` folder in your working directory. The test file should have the same name as the test subject file.
+
+```
++-- test
+|   +-- bar.js
++-- bar.js
+```
+
+Only test files (with at least one test case) should live directly under `/test`. If you need to add external mocks or fixtures, place them in a sub folder, for example:
```

Comment:
Not sure I understand the first sentence here, maybe we should just remove it?

Reply:
When you put a non-test file which matches test discovery pattern, e.g: `editor/test/random.js` and it doesn't contain any tests, Jest will throw an error for this file. That's why they should be located in subfolder to avoid those issues, e.g. `editor/test/helpers/random.js`. Any ideas how to reword?

Reply:
I think this part is misleading `(with at least one test case)` but now that you explained it, it clear enough :).